components/__snapshots__/GiveStarCard.test.js.snap

```diff
@@ -0,0 +1,43 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`GiveStarCard Component should display \`already given star to\` display when starGiven prop does not equal empty string 1`] = `
+<div
+  aria-hidden="true"
+/>
```

Comment:
I think the snapshots weren't generated correctly because every snap only has a div in it.

Look at the [curriculum page snap](https://github.com/garageScript/c0d3-app/blob/master/__tests__/pages/__snapshots__/curriculum.test.js.snap) for example, it contains the entire rendered component.

If this is incorrect it may lead to a false believe that the component is working as intended, when in reality the test is not doing anything.

Reply:
Ah, thanks for the catch!